source/util/FileUtils.cpp

```diff
@@ -302,3 +302,27 @@ bool FileUtils::FileExists(const string &filename)
     ifstream f(expandedPath);
     return f.good();
 }
+
+bool FileUtils::IsValidFilePath(const string &filePath)
+{
```

Comment:
I would suggest adding comments in this method so you do not have to write comments in code wherever we call `IsValidFilePath` method.